apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/layout.tsx

```diff
@@ -71,22 +71,17 @@ export default async function RootLayout({
             />
           }
           purchaseDetails={
-            cms.defaultPurchase.data.attributes.purchaseDetails.data.attributes.localizations.data.at(
-              0
-            )?.attributes ||
-            cms.defaultPurchase.data.attributes.purchaseDetails.data.attributes
+            cms.defaultPurchase.purchaseDetails.localizations.at(0) ||
+            cms.defaultPurchase.purchaseDetails
```

Comment:
Just a question, no changes needed. Was it considered moving this logic to the `util` classes, e.g. `PurchaseWithDetailsOfferingContentUtil`?

Definitely not asking for any changes to this as part of this PR.

Reply:
It's certainly possible to do that

Reply:
Just some additional context, currently the `util` classes already massage the data received from the CMS so that's in a more easily consumable format. 

Could the same be done for the localizations data as well, so that the `util` class only returns the data for the requested`locale`.